ci/packages/suite-web.yml

```diff
@@ -41,8 +48,24 @@ suite-web build stable:
       - releases
       - schedules
       - /^release\//
+  script:
+    - yarn install --frozen-lockfile --cache-folder .yarn --prefer-offline
+    - yarn build:libs
+    - assetPrefix=/web yarn workspace @trezor/suite-web build
+  artifacts:
+    expire_in: 7 days
+    paths:
+      - packages/suite-web/scripts/s3sync.sh
+      - packages/suite-web/build
+
+suite-web build stable codesign:
+  stage: build
+  <<: *config_sign_stable
+  only:
+    refs:
       - codesign
   script:
+    - export JWS_PUBLIC_KEY="$jws_public_key"
```

Comment:
Can't we set the `JWS_PUBLIC_KEY` env var directly without this reassigment?

Reply:
Right, addressed in 55f6ce9

Reply:
Thx! Resolving.